index.js

```diff
@@ -3,13 +3,15 @@
 var Promise = require('bluebird');
 var once = require('./lib/once');
 
-module.exports = function(app) {
+
+module.exports = function (options) {
+
   var initCb;
   var init = Promise.fromCallback(function(cb) { initCb = once(cb); });
+  var readyKey = options.key || 'ready';
```

Comment:
You probably want to make sure options is set. Maybe use the `object-assign` module to set the defaults:

```
var assign = require('object-assign');
...
  options = assign({
    key: 'ready',
  }, options);
```